components/formats-bsd/src/loci/formats/out/TiffWriter.java

```diff
@@ -343,6 +343,9 @@ public void close() throws IOException {
     if (in != null) {
       in.close();
     }
+    if (tiffSaver != null) {
+      tiffSaver.close();
```

Comment:
probably in another PR. Since it will require a review of many more readers/writers